images/build/cross/Makefile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY:	build push
+
+STAGING_REGISTRY?=gcr.io/k8s-staging-build-image
+PROD_REGISTRY?=us.gcr.io/k8s-artifacts-prod/build-image
+IMAGE=kube-cross
+
+TAG?=kubernetes-$(shell git describe --tags --match='v*' --abbrev=14)
+KUBE_CROSS_VERSION=$(shell cat VERSION)
+
+all: build push
+
+build:
+	docker build \
+		-t $(STAGING_REGISTRY)/$(IMAGE):$(TAG) \
+		-t $(STAGING_REGISTRY)/$(IMAGE):$(KUBE_CROSS_VERSION) \
+		-t $(PROD_REGISTRY)/$(IMAGE):$(KUBE_CROSS_VERSION) \
+		.
+
+push:
+	docker push $(STAGING_REGISTRY)/$(IMAGE):$(TAG)
```

Comment:
Let's please add a manifest with a single arch (amd64) for now.

```
	docker push $(STAGING_REGISTRY)/$(IMAGE)-$(ARCH):$(TAG)
	docker push $(STAGING_REGISTRY)/$(IMAGE)-$(ARCH):$(KUBE_CROSS_VERSION)
	docker manifest annotate $(STAGING_REGISTRY)/$(IMAGE) $(STAGING_REGISTRY)/$(IMAGE)-$(ARCH) --arch amd64
	docker manifest push -p $(STAGING_REGISTRY)/$(IMAGE)
```

This will help folks who end up with issues like https://github.com/kubernetes/kubernetes/issues/78964 and will give us a path to add other arch(es) in the future to support https://github.com/kubernetes/kubernetes/issues/75114 for example.

Reply:
Thanks @dims! Added.